ros_buildfarm/common.py

```diff
@@ -267,16 +267,19 @@ def get_short_os_name(os_name):
 
 def get_short_os_code_name(os_code_name):
     os_code_name_mappings = {
-        'jessie': 'J',
         'saucy': 'S',
-        'stretch': 'S',
         'trusty': 'T',
         'utopic': 'U',
         'vivid': 'V',
         'wily': 'W',
         'xenial': 'X',
         'yakkety': 'Y',
         'zesty': 'Z',
+        'artful', : 'A',
+        'bionic', : 'B',
+        'wheezy', : 'W',
+        'jessie': 'J',
```

Comment:
I think it makes sense to keep things in alphabetical order. It's not enforced here, but is what we do elsewhere.

Reply:
I don't have overridingly strong preferences but I do prefer chronological rather than alphabetical sorting for distro codenames.

Reply:
The code currently uses an alphabetical order. When adding new distros that should stay that way.

Reordering should be done separately (which I agree with @tfoote is not desired imo).

Reply:
While I really don't care all that much, I will point out that other places in ros-infrastructure do use Ubuntu-release-order, Debian-release-order: https://github.com/ros-infrastructure/buildfarm_deployment_config/blob/master/hiera/hieradata/buildfarm_role/repo.yaml#L100, https://github.com/ros-infrastructure/rospkg/blob/master/stdeb.cfg#L7 (and the rest of the packages with stdeb.cfg).

That being said, I will change this back to the original order to get this moving.